setup.cfg

```diff
@@ -18,6 +18,26 @@ package_dir =
     johnny = johnny/base
 packages = find:
 python_requires = >=3.6
+install_requires =
+    click~=8.0.1
+    Flask~=2.0.1
+    cycler~=0.10.0
+    decorator~=4.4.2
+    itsdangerous~=2.0.1
+    Jinja2~=3.0.1
+    kiwisolver~=1.3.1
+    MarkupSafe~=2.0.1
+    matplotlib~=3.4.2
+    more-itertools~=8.8.0
+    networkx~=2.5.1
+    numpy~=1.20.3
+    petl~=1.7.4
+    Pillow~=8.2.0
+    protobuf~=3.17.3
+    pyparsing~=2.4.7
+    python-dateutil~=2.8.1
+    six~=1.16.0
+    Werkzeug~=2.0.1
```

Comment:
Pinning the revision of the packages in the `setup.cfg` is not recommended, you should just specify things like minimum required versions. A reproducible deployment is usually configured through a pip's `requirements.txt` or equivalent for other Python package managers. Also, usually transitive dependencies are not listed in `setup.cfg`.